src/hyperswarm-dht-logger.js

```diff
@@ -50,25 +52,29 @@ class HyperSwarmDHTLogger {
 
     this.fileWatcher = new Tail(this.file)
     this.fileWatcher.on('line', (line) => {
-      this.sockets.forEach(socket => {
+      Array.from(this.sockets.values()).forEach(socket => {
```

Comment:
I would suggest in this case to simply iterate over values with `for of` loop since Array.from already makes an iteration and we would have 2 loops with current solution. On other case a simple for of would have just a single loop and performance on tail should be really important